lib/features/steps/runner_steps.rb

```diff
@@ -64,6 +64,23 @@
   Docker.start_service(service, command: one_line_cmd)
 end
 
+# Allows validation of the last exit code of the last run docker-compose command.
+# Will fail if no commands have been run.
+#
+# @step_input expected_code [Integer] The expected exit code
+Then('the exit code of the last docker command was {int}') do |expected_code|
```

Comment:
I think it'd be nice to have an unsuccessful version as well; I don't think the exact exit code matters very often, but it's useful to check it's not 0